src/main/java/uk/gov/justice/laa/portal/landingpage/service/UserService.java

```diff
@@ -64,7 +63,14 @@ public User createUser(String username, String password) {
         passwordProfile.setForceChangePasswordNextSignIn(true);
         passwordProfile.setPassword(password);
         user.setPasswordProfile(passwordProfile);
-        return graphClient.users().post(user);
+        User saved = graphClient.users().post(user);
+        UserModel userModel = new UserModel();
+        userModel.setEmail(user.getDisplayName());
+        userModel.setPassword("NotSave");
```

Comment:
What is the use case for passwordProfile.password vs UserModel.password?
Would these not be the same?

Reply:
the user model has not been defined yet. but we are not going to save password in our database. I put set password here just because now password field is not null